DeAceroDLabs/marketplace-frontend
# Fix/dmp 690 real data on cart items

## What is done

[[DMP-690] real data on cartitems](https://deacero.atlassian.net/browse/DMP-690)

## How is done

- Added CRUD to shoppingCartItems.
- When clicking to shoppingCart view you should see the prices updated.

## How to test

1. Follow steps on: https://github.com/DeAceroDLabs/mkt-place-back/pull/31
2. Run the project via `npm start`.
3. Add product and see how it is added on database:
![image](https://user-images.githubusercontent.com/81311646/202827695-6057ebb6-6141-4944-bba2-6af1c2f71b57.png)

5. clear localstorage and refresh page, you should see your product anyways.
6. Add the same product twice, and see how quantity is updated also on the DB.
7. Delete some product and see how it also deleted from db.
8. Change user and cartitems should change.
